lib/diagnostic.rb

```diff
@@ -108,10 +110,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = { :favorite_number => 7, :first_name => "Zelda"}'
```

Comment:
Good we can shorten this even further like this:

`person = { favorite_number: 7, first_name: 'Zelda'}`